src/utils.ts

```diff
@@ -8,6 +8,13 @@ export function getOrKey(key: string, obj: { [s: string]: string }): string {
 export const delay = (time: number) =>
   new Promise(resolve => setTimeout(resolve, time));
 
+export function fixCredit(minUnits: number, maxUnits: number) : string {
+  if(minUnits > 0) {
+    return minUnits + " - " + maxUnits
```

Comment:
Purely optional but I tend to prefer template strings: 

`` `${minUnits} - ${maxUnits}` ``

Reply:
Yeah, my b. I do think it's nicer. Just not used to doing so but I changed it